datahub-web-react/src/app/entityV2/shared/tabs/Incident/AcrylComponents/IncidentDrawerHeader.tsx

```diff
@@ -2,24 +2,25 @@ import React from 'react';
 import { Link as LinkIcon, PencilSimpleLine, X } from '@phosphor-icons/react';
 import styled from 'styled-components';
 import { Tooltip2 } from '@src/alchemy-components/components/Tooltip2';
+import { EntityPrivileges } from '@src/types.generated';
 import { useIncidentURNCopyLink } from '../hooks';
-import { IncidentAction } from '../constant';
+import { IncidentAction, noPermissionsMessage } from '../constant';
 import { IncidentTableRow } from '../types';
 import { StyledHeader, StyledHeaderActions, StyledTitle } from './styledComponents';
 
-const EditButton = styled(PencilSimpleLine)`
+const EditIcon = styled(PencilSimpleLine)`
```

Comment:
You can also try to use our button component as a wrapper

// Option 1: 
<Button icon={icon} disabled={!canEditIncidents} /> 

// Option 2: 
<Button disabled={!canEditIncidents}>
    {icon} 
</Button>